src/gui/addnewtorrentdialog.ui

```diff
@@ -203,22 +203,61 @@
               </item>
               <item>
                <layout class="QGridLayout" name="gridLayout">
-                <item row="2" column="0">
-                 <widget class="QCheckBox" name="doNotDeleteTorrentCheckBox">
-                  <property name="toolTip">
-                   <string>When checked, the .torrent file will not be deleted regardless of the settings at the &quot;Download&quot; page of the Options dialog</string>
-                  </property>
+               <item row="0" column="0">
+                 <widget class="QCheckBox" name="startTorrentCheckBox">
                   <property name="text">
-                   <string>Do not delete .torrent file</string>
+                   <string>Start torrent</string>
                   </property>
                  </widget>
                 </item>
-                <item row="1" column="1">
-                 <widget class="QCheckBox" name="firstLastCheckBox">
+                <item row="0" column="1">
+                 <layout class="QHBoxLayout" name="stopConditionLayout">
+                 <item>
+                 <widget class="QLabel" name="stopConditionLabel">
                   <property name="text">
-                   <string>Download first and last pieces first</string>
+                   <string>Stop condition:</string>
                   </property>
                  </widget>
+                 </item>
+                 <item>
+                 <widget class="QComboBox" name="stopConditionComboBox">
+                  <property name="currentIndex">
+                   <number>0</number>
+                  </property>
+                  <property name="toolTip">
+                   <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-weight:700;&quot;&gt;None - &lt;/span&gt;No stop condition is set.&lt;/p&gt;&lt;p&gt;&lt;span style=&quot; font-weight:700;&quot;&gt;Metadata received&lt;/span&gt; - Torrent will stop after metadata is received. &lt;span style=&quot; font-style:italic;&quot;&gt;Torrents that have metadata initially aren't affected.&lt;/span&gt;&lt;/p&gt;&lt;p&gt;&lt;span style=&quot; font-weight:700;&quot;&gt;Files checked&lt;/span&gt; - Torrent will stop after files are initially checked. &lt;span style=&quot; font-style:italic;&quot;&gt;This will also download metadata if it wasn't there initially.&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
```

Comment:
Personally I would prefer plain text (without special text formatting), hopefully that makes the translator less likely to mess the formatting up.
I don't strongly insist on it, do as you will.

Reply:
> hopefully that makes the translator less likely to mess the formatting up.

I can move tooltip assignment into c++ code and separate text from html.

Reply:
>I can move tooltip assignment into c++ code and separate text from html.

At least it will be better than current.

Reply:
> > I can move tooltip assignment into c++ code and separate text from html.
> 
> At least it will be better than current.

So you still think plain text tooltip is unconditionally better than rich text one, right? I am also a supporter of simplicity in many aspects. But this tooltip looks poorly readable to me without rich formatting. 

(Initially I thought it was possible to set separate tooltips for each item of combo box.)

Reply:
>So you still think plain text tooltip is unconditionally better than rich text one, right?

A little extreme assumption, aren't you?

>I am also a supporter of simplicity in many aspects. But this tooltip looks poorly readable to me without rich formatting.

Since translator factor is a consideration here, I just think it isn't worth the risk and rather keep it less error-prone for everyone. As for readability, the user will only experience it once so I'm willing to neglect the aesthetic.

Reply:
> > So you still think plain text tooltip is unconditionally better than rich text one, right?
> 
> A little extreme assumption, aren't you?

I drew this conclusion from your answer above. Since the elimination of problems with the translation does not make rich text tooltip good for you, and I did not see any other conditions in your previous comment, it turns out that plain text tooltip is unconditionally better for you.

Reply:
> > I can move tooltip assignment into c++ code and separate text from html.
> 
> At least it will be better than current.

Done.